kairon/api/app/routers/bot/channels.py

```
@@ -32,6 +33,15 @@ async def add_channel_config(
     )
     return Response(message='Channel added', data=channel_endpoint)
 
+@router.post('/mail/exists', response_model=Response)
+async def check_mail_exists(
+        request_data: ChannelRequest,
+        current_user: User = Security(Authentication.get_current_user_and_bot, scopes=DESIGNER_ACCESS)
```

Ruff (0.8.2)

Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)